.github/workflows/cypress.yml

```diff
@@ -79,3 +79,96 @@ jobs:
         with:
           name: cypress-screenshot-artifacts
           path: cypress/screenshots
+
+      - name: Archive Cypress Mochawesome test results
+        uses: actions/upload-artifact@v2
+        if: ${{ always() }}
+        with:
+          name: cypress-mochawesome-test-results-artifact-${{ matrix.ci_node_index }}
+          path: cypress/results
+
+  mochawesome-report:
+    if: ${{ always() }}
```

Comment:
Do we always want to run this, or should this job only be running on merge to master?

Reply:
We're going to start by always running it, so we have data about failing tests.